components/Header/Header.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return (
+    <>
+      <StyledHeader>TORTUGA</StyledHeader>
+    </>
+  );
+}
+
+const StyledHeader = styled.h1`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 59px;
+  text-align: center;
+  justify-content: center;
```

Comment:
```suggestion
```

`justify-content` has no effect on this element since it’s neither a flex container nor a grid container.

Reply:
Done.